docs/workflow/advanced-contributing.md

```diff
@@ -72,13 +72,12 @@ The following provides greater detail on the process:
 
 2. Review existing issues and proposals to make sure no other user is proposing a similar idea.
 
-    The design proposals are <a
-    href="https://github.com/docker/docker/pulls?q=is%3Aopen+is%3Apr+label%
-    3Akind%2Fproposal" target="_blank">all online in our GitHub pull requests</a>.
+    The design proposals are [all online in our GitHub pull requests]
+    (https://github.com/docker/docker/pulls?q=is%3Aopen+is%3Apr+label%3Akind%2Fproposal).
 
 3. Talk to the community about your idea.
 
-    We have lots of <a href="../get-help/" target="_blank">community forums</a>
+    We have lots of [community forums](/opensource/get-help.md) 
```

Comment:
I think this should be `../get-help.md` so that it works both on docs.docker.com and on github